velox/exec/Task.cpp

```diff
@@ -2765,6 +2811,17 @@ std::shared_ptr<ExchangeClient> Task::getExchangeClientLocked(
   return exchangeClients_[pipelineId];
 }
 
+void Task::maybeWriteQueryMetadata() {
+  // TODO: Make it unlikely.
+  if (queryTraceConfig_) {
+    traceDirectory_ = queryTraceConfig_->queryTraceDir;
+    const auto traceMetaDir = createTraceDirectory("metadata");
```

Comment:
```
s/metadata/config/
```

Reply:
We write not only query configs but also query nodes.